RadixWallet/Features/SettingsFeature/AccountSecurity/ManualAccountRecoveryScan/ManualAccountRecoverySeedPhrase+View.swift

```diff
@@ -105,17 +99,17 @@ private extension View {
 private extension ManualAccountRecoverySeedPhrase.View {
 	private func subtitle(isOlympia: Bool) -> String {
 		if isOlympia {
-			"Choose the Olympia seed phrase to use for derivation" // FIXME: Strings
+			"Choose the \"Legacy\" Olympia seed phrase for use for derivation:" // FIXME: Strings
```

Comment:
best check with Matt if we wants the standad quatation mark like here (requires escaping) or if he likes the full width one, see above:

```
"Tap “I’m a New Wallet User”. After...."
```